examples/fluids/meshes/cylinder.geo

```diff
@@ -0,0 +1,154 @@
+// Created Thursday, October 13, 2022.
+// Number of elements
+N = 40;
+Nx1 = N/2 + 1; Rx1 = 1.00;
+Nx2 = N/2 + 1; Rx2 = 1.00;
+Ny  = N/2 + 1; Ry = 2.00;
+Nb  = N/2 + 1; Rb = 0.9;
+Nc  = N/2 + 1; Rc = 1.00;
+
+zspan = 0.2;
+
+DefineConstant[
+  H = {4.5, Min .1, Max 4.5, Step .1,
+    Name "Parameters/Height"}
+];
+
+DefineConstant[
+  xL = {4.5, Min .1, Max 4.5, Step .1,
+    Name "Parameters/XL"}
+];
+
+DefineConstant[
+  xR = {15.5, Min .1, Max 15.5, Step .1,
+    Name "Parameters/XR"}
+];
+
+DefineConstant[
+  cyldiameter = {1.41421356, Min .1, Max 1.41421356, Step .1,
```

Comment:
Please fix up how this value is used. It is not the diameter as currently expressed. The default will be 1 once you fix how it gets used.